Assets/MixedRealityToolkit.Providers/OpenVR/GenericOpenVRController.cs

```diff
@@ -264,7 +264,12 @@ protected override bool TryRenderControllerModel(Type controllerType, InputSourc
                 }
 
                 OpenVRRenderModel openVRRenderModel = controllerModelGameObject.AddComponent<OpenVRRenderModel>();
-                openVRRenderModel.shader = visualizationProfile.GetDefaultControllerModelMaterialOverride(GetType(), ControllerHandedness).shader;
+                Material overrideMaterial = visualizationProfile.GetDefaultControllerModelMaterialOverride(GetType(), ControllerHandedness);
+                if (overrideMaterial != null)
```

Comment:
If nothing is set in the profiles (either for the global override or a platform-specific override), what does this end up using as the shader for the render model? Does it default to Unity's Standard shader, or the magenta coloring?

Reply:
It is set to the MRTK Standard wherever openVRRenderModel.shader is used, by checking it for null first

Reply:
Ahh good catch. I see it now. Thanks!